projects/plugins/boost/app/features/image-guide/src/initialize.ts

```diff
@@ -31,8 +30,22 @@ function getClosestContainingAncestor( node: HTMLElement ): HTMLElement | null {
 			( style.position === 'static' || style.position === 'relative' )
 		) {
 			elementsWithoutZIndex.push( current );
+		}
+
+		// Ok. This works, but by sheer luck. Not a good solution.
+		else if ( style.position !== 'absolute' ) {
+			elementsWithoutZIndex = [ current ];
 		} else {
-			elementsWithoutZIndex = [];
+			// The idea here was to reset elementsWithoutZIndex array
+			// whenever we encounter an element that has a z-index set.
+			// But that breaks the portfolio in the portfolio plugin layout.
+			// By not resetting the array, we're keeping
+			// tracks of first static element in the tree.
+			// This seems to work out ok for now,
+			// But at this point, the logic here now
+			// has changed from "reset the array when we encounter a z-indexed element"
+			// to "reset when we encounter a non-static element that's absolutely positioned"
+			// elementsWithoutZIndex = [];
```

Comment:
`@TODO`

Reply:
This has been done in d1f6125 and 9103e82.